metisp/pymetis/src/pymetis/classes/dataitems/dataitem.py

```diff
@@ -305,14 +402,22 @@ def add_properties(self) -> None:
         Currently only adds ESO PRO CATG to every product,
         but derived classes are more than welcome to add their own stuff.
         Do not forget to call super().add_properties() then.
+
+        #ToDo: this should not be called for raws, those do not have a PRO CATG.
         """
-        self.properties.append(
-            cpl.core.Property(
-                "ESO PRO CATG",
-                cpl.core.Type.STRING,
-                self.name(),
+        if self.frame_group() == cpl.ui.Frame.FrameGroup.RAW:
```

Comment:
The `RAW` from the `cpl.ui.Frame.FrameGroup` has nothing to do with the data being RAW data.

The `RAW` just denotes the primary input of the recipe, which might be RAW data, but it might also be processed data.

But if it is input data, then it should already have a `PRO.CATG` defined, so this check also does not do any harm.

So it seems fine, but a bit smelly.